### DISCUSSION TITLE
Applicable Reader

### DISCUSSION THREAD

**Discussion by zieb-a** [scs/smartmeter-datacollector#20] [Q&A] [answered]
For the Landis+Gyr e450, it appears that an optical or wireless M-Bus interface is available, however the documentation mentions wired M-Bus, which does not seem to be available. Which hardware is required to read the data?

**Comment by raymar9** [comment, marked as answer]
With the Landis+Gyr E450 you can only use the wired M-Bus interface which must be enabled to push out meter data. Please check our [Wiki](https://github.com/scs/smartmeter-datacollector/wiki). There you find a suitable M-Bus to USB adapter, however others have successfully used alternative (cheaper) adapters.
